autoware_launch/config/control/trajectory_follower/longitudinal/pid.param.yaml

```diff
@@ -53,7 +53,7 @@
 
     # stopped state
     stopped_vel: 0.0
-    stopped_acc: -3.4 # denotes pedal position
+    stopped_acc: -2.49 # denotes pedal position
```

Comment:
stopped_acc does not denote deceleration rate, but denotes pedal position value after the velocity becomes zero.